core/commands/object.go

```diff
@@ -344,39 +344,21 @@ Data should be in the format specified by the --inputenc flag.
 }
 
 // objectData takes a key string and writes out the raw bytes of that node (if there is one)
-func objectData(n *core.IpfsNode, fpath path.Path) (io.Reader, error) {
-	dagnode, err := core.Resolve(n, fpath)
+func objectData(n *dag.Node, err error) (io.Reader, error) {
```

Comment:
There's so little left in this function after you remove the `Resolve` call that I'd just drop this (and `objectLinks`, etc.) entirely.  Unless we really want to keep them around for the debug logging…